ratis-server-api/src/main/java/org/apache/ratis/statemachine/TransactionContext.java

```diff
@@ -60,7 +60,10 @@ public interface TransactionContext {
   /**
    * Returns the data from the {@link StateMachine}
    * @return the data from the {@link StateMachine}
+   * @deprecated access StateMachineLogEntry via @{@link TransactionContext#getLogEntryRef()} or
+   * @{@link TransactionContext#getLogEntryUnsafe}
```

Comment:
Typos:
- `@{@link ...}` -> `{@link ...}` for both links.
- The second link needs `()`.
```java
   * @deprecated access StateMachineLogEntry via {@link TransactionContext#getLogEntryRef()}
   *             or {@link TransactionContext#getLogEntryUnsafe()}
```